astroid/nodes.py

```diff
@@ -189,6 +194,8 @@
     With,
     Yield,
     YieldFrom,
-    FormattedValue,
-    JoinedStr,
 )
+
+# Can't create a proper __all__ with string because of a cyclic import for ClassDef
+__all__ = ["CONST_CLS"]
+__all__ += [c.__name__ for c in ALL_NODE_CLASSES]
```

Comment:
I'm still not convinced this changes anything

Reply:
This clarify the API and also let linter see that the import is actually used (autoflake8 remove unused import)

Reply:
Also it's necessary if we want to be able to keep the old API and deprecate it in #1057 :
 ```
 +# It's impossible to import from astroid.nodes with a wildcard, because
 +# there is a cyclic import that prevent creating an __all__ in astroid/nodes
 +# and we need astroid/scoped_nodes and astroid/node_classes to work. So
 +# importing with a wildcard would clash with astroid/nodes/scoped_nodes
 +# and astroid/nodes/node_classes.
```